metrics/metrics.go

```diff
@@ -45,6 +45,12 @@ func (m *LabelMap) Get(key string) *expvar.Int {
 	return m.Map.Get(key).(*expvar.Int)
 }
 
+// GetIncrFunc returns a function that increments the expvar.Int named by key.
+func (m *LabelMap) GetIncrFunc(key string) func(delta int64) {
+	v := m.Get(key)
+	return func(delta int64) { v.Add(delta) }
```

Comment:
Would it suffice to return v.Add directly rather than a separate closure?

Reply:
It would indeed and I'm surprised I didn't! Probably because I'd just written the other implemention which couldn't.